src/definitions/modules/dimmer.less

```diff
@@ -63,7 +63,7 @@
 
 
 /* Loose Coupling */
-.ui.segment > .ui.dimmer {
+.ui.segment > .ui.dimmer:not(.page) {
```

Comment:
What is a motivation to this change?

Reply:
When the dimmer is inside a segment and we add the page class to extend it on the screen, it inherits the border radius property of the segment, which is unpleasant. Once extended to the page it must be fully overlaid on the page.